sktime/benchmarking/base.py

```diff
@@ -182,7 +183,10 @@ def __init__(self, strategy_name, dataset_name, index, y_true, y_pred,
         self.y_true = y_true
         self.y_pred = y_pred
         self.y_proba = y_proba
-
+        self.fit_estimator_start_time = fit_estimator_start_time
```

Comment:
saving the fit and predict time is probably enough, or are the absolute start and end time points interesting in some cases?

Reply:
For my specific use case I don't currently need the absolute start and end time points. However, my thinking was that:

1. We have the data so why not save it in case it might be useful in the future. It is not a big drag on resources, especially if results are saved in RAM. If results are saved on HDD, my current implementation is not ideal as it saves an entire column of the same copy of the values which is wasteful. However, I have not found a better way to persist the data but will investigate further.
2. The design is such that the orchestrator saves the data and the data is subsequently processed by the evaluator. It is true that taking the difference of two numbers is not a very complicated calculation so it is totally possible to do this in the orchestrator but this still deviates a little bit from the overall design logic.